tests/unit/test_preprocessing.py

```diff
@@ -23,3 +24,11 @@ def test_loner_removal(self):
         num_vertices = len(g.vertices)
         g = preprocessing.remove_loners(g)
         self.assertEqual(num_vertices - 1, len(g.vertices))
+
+    def test_use_complement(self):
```

Comment:
Test ook of een graaf met klein aantal edges wel zichzelf teruggeeft bij deze check.